package.json

```diff
@@ -14,7 +14,9 @@
     "build:browser": "tsc --project tsconfig.json --module ESNext --outDir ./browser",
     "build:main": "tsc --project tsconfig.json --module CommonJS --outDir ./build",
     "build:tests": "tsc --project tsconfig.json --module CommonJS --outDir ./test-app/remix-validated-form",
-    "test": "jest src",
+    "build:sample": "tsc --project tsconfig.json --module CommonJS --outDir ./sample-app/remix-validated-form",
+    "sample-app": "npm run build:sample && cd sample-app && npm run dev",
+    "test": "jest src --watch",
```

Comment:
This _might_ break CI, since this is run as part of the github action. We could add a separate `test:watch` script though.

Reply:
Cool. `test-watch` added.